src/shared-components/dropdown/style.ts

```diff
@@ -76,7 +82,10 @@ export const IconContainer = styled.div`
   }
 `;
 
-export const DropdownOptionsContainer = styled.ul`
+export const DropdownOptionsContainer = styled.ul<{
+  isOpen: boolean;
+  optionsContainerMaxHeight?: string;
```

Comment:
This is one of the downstream optional types that is actually always provided (currently, via `defaultProps`).